WindowsInstaller/config_template.txt

```diff
@@ -186,3 +186,19 @@ room_information_dialogs = true
 -- accessible, the game will disallow any attempt to blocking the path.
 allow_blocking_off_areas = false
 -------------------------------------------------------------------------------
+
+-------------------------------------------------------------------------------
+-- Direct Zoom: An experimental enhancement to avoid rendering to an
+-- intermediate texture when zooming. Improves performance and reliability on
+-- some hardware.
+direct_zoom = nil
+-------------------------------------------------------------------------------
+
+-------------------------------- CAMPAIGN MENU --------------------------------
+-- By default your computer log in will be your name in the game.  You can change
+-- it in the campaign menu or between the brace brackets below [[YOUR NAME]].
+-- Note: space is limited in the game, so don't enter a name that is too long!
+--
+-- If you wish to go back to the default settings for everything, you can delete
```

Comment:
This doesn't look related to the campaign menu and thus should be clearly separated from it.